src/surfaces/actionBar/actionBarActionsButtonDrawerOption.jsx

```diff
@@ -79,13 +86,13 @@ class ActionBarActionsButtonDrawerOption extends React.PureComponent {
             `${hide ? `actions_button_drawer--option_container-${idNumber}-hide` : ''}`,
             `${!hide && !isSelected ? `actions_button_drawer--option_container-${idNumber}-show` : ''}`,
             {
-                'actions_button_drawer--option_container-selected': isSelected,
+                [`actions_button_drawer--option_container-${idNumber}-selected`]: isSelected,
```

Comment:
If `idNumber` is `null` is it okay if these end up as `actions_button_drawer--option_container--show` and `actions_button_drawer--option_container--selected`?

Reply:
Fixed. Default value is `1` now.

Reply:
Sounds good.
Also should `option_container-${idNumber}` be `option_container_${idNumber}`?
Like, is all of that the "element", and `selected` is the "modifier"?

Reply:
No, the id and `selected` are the modifiers.

Reply:
Very well.  I'll buy what you're selling.  We can resolve this thread if you like.